Apps/PatientDataAccess/src/ConfigurationExtentions.cs

```diff
@@ -0,0 +1,53 @@
+// -------------------------------------------------------------------------
+//  Copyright © 2019 Province of British Columbia
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+// -------------------------------------------------------------------------
+
+using System;
+using HealthGateway.Common.Utils.Phsa;
+using HealthGateway.PatientDataAccess.Api;
+using Refit;
+
+namespace HealthGateway.PatientDataAccess
+{
+    using Microsoft.Extensions.DependencyInjection;
+
+    /// <summary>
+    /// Helper class to add and configure <see cref="IPatientDataRepository"/> dependencies
+    /// </summary>
+    public static class ConfigurationExtentions
```

Comment:
Typo in class & file name: should be "ConfigurationExtensions".

Reply:
fixed